src/app/components/ManualEntry.svelte

```diff
@@ -1,9 +1,57 @@
 <script lang="ts">
-  /**
-   * Handles the click when the delete button is pressed
-   * TODO
-   */
-  const deleteRow = (): void => {};
+  import Button from "./Button.svelte";
+  import worker from "../spawn-worker";
+
+  interface rowDataObj {
+    word: string;
+    count: number | undefined;
+  }
+
+  export let tableData: {
+    name: string;
+    data: rowDataObj[];
+  };
+
+  let rowDataFromManualEntry = tableData.data;
+  let columns = [
+    {
+      label: "Word",
+      name: "word",
+    },
+    {
+      label: "Count",
+      name: "count",
+    },
+    {
+      label: "Action",
+      name: "action",
+    },
+  ];
+  $: validInput = rowDataFromManualEntry.every((element) =>
+    validateRowData(element)
+  );
+  $: btnColor = validInput ? "blue" : "grey";
+
+  const validateRowData = (rowData: rowDataObj): boolean => {
+    const word = rowData.word;
+    return word !== "" && word !== undefined && word !== null;
```

Comment:
Please add a check that word is not just whitespace! e.g., `"  \t \n"` is not a word in any language!